src/collectors/portstat/portstat.py

```diff
@@ -0,0 +1,69 @@
+
+"""
+The PortStatCollector collects metrics about ports listed in config file.
+
+##### Dependencies
+
+* psutil
+
+"""
+
+from collections import Counter
+
+import psutil
+import diamond.collector
+
+
+def get_port_stats(port):
```

Comment:
Wouldn't it be cleaner to "hide" that inside the collector class? Just asking ;)